phone/features/onboarding/src/main/java/jp/ac/mayoi/onboarding/OnboardingScreen.kt

```diff
@@ -1,2 +1,41 @@
 package jp.ac.mayoi.onboarding
 
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun kasu_mitaina_composable(
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mayoi-design/maigo_compass_android/blob/2d73b0f6dbe3f135b349348ff0001248f4b9d7a3/phone/features/onboarding/src/main/java/jp/ac/mayoi/onboarding/OnboardingScreen.kt#L16"></span>Composable functions that return Unit should start with an uppercase letter</td>
    </tr>
  </tbody>
</table>